src/plots/plots.js

```diff
@@ -811,10 +811,7 @@ plots.supplyDataDefaults = function(dataIn, dataOut, layout, fullLayout) {
                 var expandedTrace = expandedTraces[j];
                 var fullExpandedTrace = plots.supplyTraceDefaults(expandedTrace, cnt, fullLayout, i);
 
-                // The group key gets cleared. If set, pass it forward
-                if(expandedTrace._group) {
-                    fullExpandedTrace._group = expandedTrace._group;
-                }
+                relinkPrivateKeys(fullExpandedTrace, expandedTrace);
```

Comment:
Maybe add a comment here e.g.:

> relink private (i.e. underscore) keys expanded trace to full expanded trace so that transform supply-default methods can set `_` keys for future use.